src/server/scripts/Northrend/Ulduar/Ulduar/boss_kologarn.cpp

```diff
@@ -223,7 +223,7 @@ class boss_kologarn : public CreatureScript
                         break;
                     case NPC_RUBBLE:
                         summons.Summon(summon);
-                        // absence of break intended
+                        [[fallthrough]];
```

Comment:
isnt this better to be includes in #25054 ?

Reply:
True. I was just lazy :) Let's fix that.

Reply:
Fixed.